src/pages/ShoppingList.js

```diff
@@ -191,16 +204,17 @@ const ShoppingList = ({ token }) => {
                 onChange={e => setFilterString(e.target.value)}
             />
             <button onClick={() => setFilterString('')}>X</button>
-            <table>
+            <tbody>
                 {filterString
```

Comment:
This `tbody` should be wrapped in a `<table>` tag, too. Then you will get the table styles back:
![image](https://user-images.githubusercontent.com/10378331/80209793-53a61b80-85f8-11ea-9baa-1e529bf8d2ca.png)

![image](https://user-images.githubusercontent.com/10378331/80209871-6b7d9f80-85f8-11ea-9c62-70b59943e579.png)